### DISCUSSION TITLE
Perses CLI

### DISCUSSION THREAD

**Discussion by agardnerIT** [perses/perses#2180] [Show and tell]
I've started creating a CLI to help users build skeleton Perses dashboards.

Happy to contribute and maintain this code in the official project if you think it'd be valuable.

Very early days so PRs and assistance much appreciated!

https://github.com/agardnerIT/perses-python-cli

**Comment by Nexucis** [comment]
From my perspective if a user is starting to generate and manipulate the JSON representing the dashboard, then he is looking for a dashboard-as-code solution. That's something we are already covering with two SDKs + the Perses CLI.

Could you clarify what use case you are foreseeing here ?

**Reply by agardnerIT** [reply]
As a new Perses user, I have an existing dashbaord JSON scheme (from a proprietary) that I want to investigate what we can do with Perses (can we migrate, can we accept Perses and translate etc.)

I just want a JSON file to compare apples to apples. The SDK is no good (for my current usecase). The DAC command requires CUE (which I have no idea about).

So really this is the "easy button".

**Comment by Nexucis** [comment]
I think AI will be the way to go for this purpose and much more powerful.
We don't intend to add more logic in the CLI to support what you are proposing.

Thank you for the suggestions you made.

I am closing this discussion for now. We may review this statement if there are more people willing to have more dialog between CLI and dashboard let say.
